src/entities.js

```diff
@@ -3,7 +3,13 @@ const list = require('./list/list')
 const entities = new Map()
 
 const builder = (entity, id) => {
-    const defaultItem = item(id)
+    const entityId = id || entity.name
```

Comment:
How does this function handle entity names with spaces? The correct thing would be to generate names without spaces and pascal case.